.github/workflows/install-and-test.yml

```diff
@@ -612,13 +660,21 @@ jobs:
         #!/usr/bin/env python3
         # -*- coding: utf-8 -*-
         #
-        # Copyright (C) 2021  The SymbiFlow Authors.
-        #
-        # Use of this source code is governed by a ISC-style
-        # license that can be found in the LICENSE file or at
-        # https://opensource.org/licenses/ISC
-        #
-        # SPDX-License-Identifier: ISC
```

Comment:
I think the indenting in this file is wrong?

Reply:
fixed